android/src/main/java/com/genesys/messenger/GenesysCloudChatActivity.kt

```diff
@@ -184,7 +184,7 @@ class GenesysCloudChatActivity : AppCompatActivity(), ChatEventListener {
 
     companion object {
         const val CONVERSATION_FRAGMENT_TAG = "conversation_fragment"
-        const val GenTag = "GenesysCloudChatActivity"
+        const val GenTag = "GenesysChatActivity"
```

Comment:
Why was this change made?

Reply:
![image](https://user-images.githubusercontent.com/33628758/145051081-843ffeaa-4dd2-4269-a9e9-8c6784d502b2.png)